export/securedrop_export/print/service.py

```diff
@@ -31,7 +31,30 @@
     "application/x-7z-compressed",
 ]
 
-MIMETYPE_PRINT_WITHOUT_CONVERSION = ["application/pdf", "text/plain"]
+# These are a subset of mimetypes that cups supports for direct printing:
+# see /usr/share/cups/mime/mime.types
+MIMETYPE_PRINT_WITHOUT_CONVERSION = [
+    "application/pdf",
+    "text/plain",
+    "image/gif",
+    "image/png",
+    "image/jpeg",
+    "image/pwg-raster",
+    "image/tiff",
+    "image/x-photocd",
+    "image/x-portable-anymap",
+    "image/x-portable-bitmap",
+    "image/x-portable-graymap",
+    "image/x-portable-pixmap",
+    "image/x-sgi-rgb",
+    "image/x-xbitmap",
+    "image/x-xpixmap",
+    "image/x-sun-raster",
+    "image/urf",
```

Comment:
This one caught my eye because I didn't recognize it and came across https://github.com/apple/cups/issues/4888#issuecomment-250948112 which suggests it isn't needed on Linux